R/format_PEW_temp.R

```diff
@@ -0,0 +1,419 @@
+#'
+#'
+#' #### --------------------------------------~
+#' #### CAPTURUE DATA                      #### in process
+#' #### --------------------------------------~
+#'
+#'
+#' # create_capture_PEW <- function(data){
+#'
+#' Capture_data <-
+#'   pew_data %>%
+#'   #### ASSUMPTION: Date is obligatory, not allowing for NAs,
+#'   # therefore we exclude observations without the Date
+#'   # (CHECK, as we may lose some information)
+#'   filter(!(is.na(Date))) %>%
+#'   #### Rename variables
+#'   dplyr::rename(CaptureDate = Date) %>%
+#'   #### Create new variables
+#'   mutate(#### ASSUMPTION: There is no captureID info, so far I use this (CHECK)
+#'          CaptureID = paste(IndvID, "CaptureNumber", sep = "_"),
+#'          #### ASSUMPTION: Some times are strange, to simplify, round to whole hour
+#'          CaptureTime = ifelse(is.na(ObservationTimeH),
+#'                               NA,
+#'                               paste(sprintf("%02d", round(ObservationTimeH, 0)),
+#'                                     "00", sep = ":")),
+#'          Sex_observed = Sex,
+#'          LocationID = NA_character_,
+#'          CaptureAlive = ifelse(Method != "Found dead", TRUE, FALSE),
+#'          #### ASSUMPTION: the same as CaptureAlive, as there is no additional data
+#'          ReleaseAlive = CaptureAlive,
+#'          CapturePopID = PopID,
+#'          CapturePlot = NA_character_,
+#'          ReleasePopID = ifelse(ReleaseAlive == TRUE, CapturePopID, NA_character_),
+#'          ReleasePlot = ifelse(ReleaseAlive == TRUE, CapturePlot, NA_character_),
+#'          #### ASSUMPTION: use Alternative, as there is no information
+#'          OriginalTarsusMethod = "Alternative",
+#'          WingLength = NA_integer_,
+#'          ExperimentID = dplyr::case_when(Experiment == "BSM - Griffioen et al. 2019 PeerJ" ~
+#'                                            "COHORT; PARENTAGE",
+#'                                          Experiment == "2h Temp D4" ~ "SURVIVAL",
+#'                                          Experiment == "2h Temp D4 + Handicaping Male: Griffioen et al. 2019 Front Ecol&Evol" ~
+#'                                            "SURVIVAL; PARENTAGE")
+#'          #### VARIABLES TO FINISH:
+#'          # Age_observed =
+#'          # Age_calculated
+#'          # ChickAge
+#'
+#'
+#'
+#'   )
+#'
+#'
+#' # #### Final arrangement
+#' # dplyr::select(CaptureID, IndvID, Species, Sex_observed, BreedingSeason,
+#' #               CaptureDate, CaptureTime, ObserverID, LocationID,
+#' #               CaptureAlive, ReleaseAlive, CapturePopID, CapturePlot,
+#' #               ReleasePopID, ReleasePlot, Mass, Tarsus, OriginalTarsusMethod,
+#' #               WingLength, Age_observed, Age_calculated, ChickAge, ExperimentID)
+#'
+#'
+#'
+#' # }
+#'
+#'
+#'
+#' #### VARIOUS ASSUMPTIONS:
+#' # For Age: - there are some idividuals with NA age, but with partner and the DateRingingChick
+#' #             >> we assume they are adults (example IndvID 12706844)
```

Comment:
Exactly how I would approach it. Generally, if there is no age specified we should assume they're adult because this is much more likely than being an unrecorded chick!